Sources/Nimble/Utils/Async.swift

```diff
@@ -1,10 +1,13 @@
+import CoreFoundation
+import Dispatch
 import Foundation
 
-#if _runtime(_ObjC)
-import Dispatch
+#if !_runtime(_ObjC)
+    private let NSEC_PER_SEC: CUnsignedLongLong = 1000000000
```

Comment:
I'd tried to replace some `TimeInterval` parameters to `DispatchTimeInterval`, but [this line](https://github.com/Quick/Nimble/blob/26f41da389a53c62b7f24ed55cd1e0e4bf665ad8/Sources/Nimble/DSL+Wait.swift#L31) made me unhappy at that time 😓